chart/compass/charts/connector/templates/tests/test-connector.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: {{ .Chart.Name }}-tests
+  labels:
+    app: {{ .Chart.Name }}-tests
+    app.kubernetes.io/name: {{ .Chart.Name }}-tests
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  disableConcurrency: true
+  template:
+    spec:
+      shareProcessNamespace: true
+      containers:
+      - name: connector-tests
```

I think the convention is that container name should be `tests`.
```suggestion
      - name: tests
```